php/blocks/class-loader.php

```diff
@@ -100,6 +100,25 @@ public function editor_assets() {
 				$this->enqueue_block_styles( $block['name'], array( 'preview', 'block' ) );
 			}
 		}
+
+		// Used to conditionally show notices for blocks belonging to an author.
+		$author_blocks = get_posts(
+			array(
+				'author'         => get_current_user_id(),
```

Comment:
Should this be `post_author` instead of `author`? Locally, only `post_author` got any posts.

Reply:
That's very weird. The `WP_Query` documentation says to use `author`. https://codex.wordpress.org/Class_Reference/WP_Query#Author_Parameters

Here's a gif of my testing using `author`. When I change one of my test blocks (the slug is `another-297-block`) so that it's authored by another user, it's not returned in the query. All my other blocks are.

![2019-05-19 10-09-45 2019-05-19 10_13_33](https://user-images.githubusercontent.com/1097667/57976244-c5580b80-7a1e-11e9-95c6-7025873d02a5.gif)

Could you please check again?

Reply:
Sorry, it looks like I missed something. The `author` key looks to work fine now:

<img width="1160" alt="author-blocks-here" src="https://user-images.githubusercontent.com/4063887/57976412-cb110b00-79a4-11e9-87bb-e19a1c5726ff.png">